src/main/java/com/example/onlinebookshop/dto/user/request/SendUserTokenRequestDto.java

```diff
@@ -0,0 +1,4 @@
+package com.example.onlinebookshop.dto.user.request;
+
+public record SendUserTokenRequestDto(String token) {
```

Comment:
Rename please. I'm not quite understand what this request is about? Where do you use it?

Reply:
Sorry, nowhere. Forgot to delete.It was a part of an idea that didn't work